receiver/k8sclusterreceiver/internal/metadata/testdata/config.yaml

```diff
@@ -59,6 +59,14 @@ all_set:
       enabled: true
     k8s.namespace.phase:
       enabled: true
+    k8s.node.allocatable.cpu:
+      enabled: true
+    k8s.node.allocatable.ephemeral_storage:
+      enabled: true
+    k8s.node.allocatable.memory:
+      enabled: true
+    k8s.node.allocatable.pods:
```

Comment:
Apologies for the back and forth here. This is going to change again with https://github.com/open-telemetry/semantic-conventions/pull/2822 because it needs to be aligned with SemConv project. I think the rest of the metrics should change too i.e. `k8s.node.cpu.allocatable` (I'll send a follow-up PR after). In this, I suggest we wait until we have the semconv part ready, which hopefully will be soon.

Reply:
thanks for taking care of it!

Reply:
ref: https://github.com/open-telemetry/semantic-conventions/pull/2973

Reply:
thanks, this PR should be adapted